test/Altinn.App.Core.Tests/Internal/Process/ServiceTasks/PdfServiceTaskTests.cs

```diff
@@ -1,178 +1,37 @@
+    private readonly Mock<IPdfService> _pdfServiceMock;
+    private readonly PdfServiceTask _serviceTask;
 
     public PdfServiceTaskTests()
     {
-        _appMetadata = new Mock<IAppMetadata>();
-        _pdfService = new Mock<IPdfService>();
-        _appModel = new Mock<IAppModel>();
-    }
-
-    [Fact]
-    public async Task Execute_calls_pdf_service()
-    {
-        Instance i = new Instance() { Data = [new DataElement() { DataType = "DataType_1" },] };
-        SetupAppMetadataWithDataTypes(
-            [
-                new DataType
-                {
-                    Id = "DataType_1",
-                    TaskId = "Task_1",
-                    AppLogic = new ApplicationLogic() { ClassRef = "DataType_1" },
-                    EnablePdfCreation = true
-                },
-            ]
-        );
-        PdfServiceTask pst = new PdfServiceTask(_appMetadata.Object, _pdfService.Object);
-        await pst.Execute("Task_1", i);
-        _appMetadata.Verify(am => am.GetApplicationMetadata(), Times.Once);
-        _pdfService.Verify(ps => ps.GenerateAndStorePdf(i, "Task_1", CancellationToken.None), Times.Once);
-        _appMetadata.VerifyNoOtherCalls();
-        _pdfService.VerifyNoOtherCalls();
-        _appModel.VerifyNoOtherCalls();
-    }
-
-    [Fact]
-    public async Task Execute_pdf_service_is_called_only_once()
-    {
-        Instance i = new Instance()
-        {
-            Data =
-            [
-                new DataElement() { DataType = "DataType_1" },
-                new DataElement() { DataType = "DataType_1" },
-                new DataElement() { DataType = "DataType_2" },
-                new DataElement() { DataType = "DataType_2" },
-            ]
-        };
-        SetupAppMetadataWithDataTypes(
-            [
-                new DataType
-                {
-                    Id = "DataType_1",
-                    TaskId = "Task_1",
-                    AppLogic = new ApplicationLogic() { ClassRef = "DataType_1" },
-                    EnablePdfCreation = true
-                },
-                new DataType
-                {
-                    Id = "DataType_2",
-                    TaskId = "Task_1",
-                    AppLogic = new ApplicationLogic() { ClassRef = "DataType_2" },
-                    EnablePdfCreation = true
-                },
-            ]
-        );
-        PdfServiceTask pst = new PdfServiceTask(_appMetadata.Object, _pdfService.Object);
-        await pst.Execute("Task_1", i);
-        _appMetadata.Verify(am => am.GetApplicationMetadata());
-        _pdfService.Verify(ps => ps.GenerateAndStorePdf(i, "Task_1", CancellationToken.None), Times.Once);
-        _appMetadata.VerifyNoOtherCalls();
-        _pdfService.VerifyNoOtherCalls();
-        _appModel.VerifyNoOtherCalls();
-    }
+        Mock<ILogger<PdfServiceTask>> loggerMock = new();
+        Mock<IProcessReader> processReaderMock = new();
+        _pdfServiceMock = new Mock<IPdfService>();
```

Comment:
Why not store all mocks as private fields and initialise them on declaration instead of in the constructor.